### PR TITLE
Version 0.16.0-beta.2

### PR DESCRIPTION
**Pull request by seanadkinson**
* Updated to version `0.16.0-beta.1
* Added changelog
* Clarified some documentation
* Documented iOS 12 crash and laser/draw issue in new Known Issues section

### CODE REVIEW

**Review by dgoldman-pdx — APPROVED** [review]
Good to see a change-log. On the changes and documentation re dependencies, scripts, etc I shall defer to your greater understanding.

**Comment by dgoldman-pdx** [comment]
Why is this `beta.1`? Why not simply `0.16.0`, which could be followed by `0.16.1` as necessary?

File: README.md
Comment (alkubaig) [inline README.md:398]: I am confused about this one?

**Review by alkubaig — APPROVED** [review]
This is Great! Thank you!

File: README.md
Comment (seanadkinson) [reply README.md:398]: Its a pretty tough issue... i did the best i could to explain, but please let me know which parts are confusing, and happy to reword.  The bug occurs under the three conditions mentioned on iOS 12 and below.

**Comment by seanadkinson** [comment]
@dgoldman-pdx I'll wait to merge this until next week, when we expect to bump the current `beta.1` to `0.16.0` proper.

File: README.md
Comment (alkubaig) [reply README.md:398]: Only the storyboard part I was confused about. 
It was not clear to me why this would cause the error. 
But maybe it is not the storyboard that causes the issue but the three combined?

Comment (seanadkinson) [reply README.md:398]: Exactly!  Above we mention that ALL the following conditions need to be met... yes, crazy bug.

**Comment by dgoldman-pdx** [comment]
> I'll wait to merge this until next week, when we expect to bump the current beta.1 to 0.16.0 proper.

That's fine, but I still don't understandd why we're adding `.beta` and `.1` when we could simply be incrementing the minor version or build number. I.e., I don't see why this release is any more "beta" than any other release we've ever published.

**Pull request merged**
